pkg/operator/ceph/controller/predicate.go

```diff
@@ -99,6 +100,7 @@ func WatchControllerPredicate() predicate.Funcs {
 				diff := cmp.Diff(objOld.Spec, objNew.Spec, resourceQtyComparer)
 				if diff != "" {
 					logger.Infof("CR has changed for %q. diff=%s", objNew.Name, diff)
+					return true
```

Comment:
@leseb this may conflict with https://github.com/rook/rook/pull/5813 IMO. Should I need to wait PR for this one to merge?

Reply:
Yes, please that'd be great. Thanks.